py_jama_rest_client/client.py

```diff
@@ -54,6 +54,12 @@ def __init__(self, host_domain, credentials=('username', 'password'), api_versio
         self.__credentials = credentials
         self.__core = Core(host_domain, credentials, api_version=api_version, oauth=oauth)
 
+    def get_available_endpoints(self):
+        """Returns a list of all the available endpoints."""
```

Comment:
Please use google style comment blocks for functions. 
EX: https://gist.github.com/dsaiztc/9b4c15c6dfef08a957c1

Reply:
doing this will allow the auto doc tools to correctly produce API documentation.